src/main/java/org/jenkinsci/plugins/p4/scm/P4SCMFile.java

```diff
@@ -137,17 +139,25 @@ protected Type type() throws IOException, InterruptedException {
 	public InputStream content() throws IOException, InterruptedException {
 		ConnectionHelper p4 = fs.getConnection();
 		List<IFileSpec> file = getFileSpec();
-
 		GetFileContentsOptions printOpts = new GetFileContentsOptions();
 		printOpts.setNoHeaderLine(true);
-
+		addJenkinsFilePathToTagAction(p4, file);
 		try {
 			return p4.getConnection().getFileContents(file, printOpts);
 		} catch (P4JavaException e) {
 			throw new IOException(e);
 		}
 	}
 
+	private void addJenkinsFilePathToTagAction(ConnectionHelper p4, List<IFileSpec> file) throws IOException, InterruptedException {
+		try {
+			List<IFileSpec> where = p4.getConnection().getCurrentClient().where(file);
```

Comment:
May not work on a Slave node (would need to be a WhereTask) alternatively you could use the local where method.

Reply:
Thanks, paul for your comment. I have used localWhere. To use this I need to get workspace root, or else p4jave will throw an exception.